prov/efa/src/rxr/rxr.h

```diff
@@ -545,6 +545,9 @@ struct rxr_ep {
 	/* Endpoint's capability to support zero-copy rx */
 	bool use_zcpy_rx;
 
+	/* Endpoint's capability to support resource management */
```

Comment:
This comment doesn't quite line up - perhaps "Application requested resource management support" instead?